.github/workflows/quality-assurance.yml

```diff
@@ -34,16 +34,16 @@ jobs:
           composer update --prefer-dist --no-interaction
 
       - name: Run tests
-        run: vendor/bin/phpunit --verbose --coverage-clover=coverage.clover
+        run: vendor/bin/phpunit --coverage-clover=coverage.clover
 
       - name: Publish code coverage
-        uses: paambaati/codeclimate-action@v2.7.5
+        uses: paambaati/codeclimate-action@v5.0.0
         env:
           CC_TEST_REPORTER_ID: ${{ secrets.CODECLIMATE_TEST_REPORTER_ID }}
         with:
+          coverageCommand: vendor/bin/phpunit --coverage-clover=coverage.clover
           coverageLocations: ${{github.workspace}}/*.clover:clover
           debug: true
 
       - name: Run PHPStan analysis
         run: vendor/bin/phpstan analyse --memory-limit=-1
-        if: ${{ matrix.php == '7.4' }}
```

Comment:
Why did you remove this? static analysis is typically done on a single version, as it is most often optimised for a single version as well.